brandingbrand/flagship
# fix(fscomponents): fix missing icon for paypal button on web

This adds missing dimension styles to the icons in the PaypalCheckoutButton component so the PayPal logo is visible on web.

**Test:**
1. Run yarn dev:storybook
2. Navigate to the PayPalCheckoutButton story
3. Verify that the PayPal logo displays in the button
